dom/observable/tentative/observable-inspect.any.js

```diff
@@ -381,32 +382,39 @@ test(() => {
   });
 
   const results = [];
+  const error = new Error("error from inspect() subscribe handler");
 
   const result = source.inspect({
     abort: () => {
-      results.push('abort() handler run');
-      throw new Error("error from inspect() subscribe handler");
+      results.push("abort() handler run");
```

Comment:
For future reference, I think it would be best to leave stylistic changes like this and `{a: b}` -> `{ a: b }` (padding) out, since they kind of clutter the PR. Just my opinion, from someone that doesn't use prettier or anything.

Reply:
Oops yeah! Totally agree, and usually do, revert changes like this. Totally dropped the ball here, sorry mate.